### PR TITLE
@l2succes: Launch gdpr ab test

### PR DESCRIPTION
**Pull request by xtina-starr**
This addresses the minor visual issue mentioned by @katarinabatina via [slack](https://artsy.slack.com/archives/C9SATFLUU/p1522765312000123) and launches the the gdpr compliance test.

### CODE REVIEW

**Comment by anipetrov** [comment]
is this including the inquiry_questionnaire modal?

**Comment by xtina-starr** [comment]
@anipetrov yes ma'am it does.

File: src/desktop/components/split_test/running_tests.coffee [hunk 1]
```diff
@@ -34,11 +34,12 @@ module.exports = {
...
-      control: 100
-      combined_checkboxes: 0
-      separated_checkboxes: 0
-    }
+    weighting: 'equal'
+    outcomes: [
+      'control'
+      'combined_checkboxes'
+      'separated_checkboxes'
+    ]
```
Comment (kanaabe) [inline src/desktop/components/split_test/running_tests.coffee]: 🎉

**Comment by anipetrov** [comment]
sorry, i meant i dont see this `splitTest('gdpr_compliance_test').view()` firing for the inquiry questionnaire case but i am probably missing something, thanks for this!

**Comment by xtina-starr** [comment]
No worries. It was already in place. You can see it [here](https://github.com/artsy/force/blob/master/src/desktop/components/inquiry_questionnaire/views/account.coffee#L34).

**Comment by anipetrov** [comment]
oh got it, thank you for explaining @xtina-starr!

**Pull request merged**